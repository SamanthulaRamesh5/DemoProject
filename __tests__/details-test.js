/**
 * @format
 */

 import 'react-native';
 import React from 'react';
import DetailesScreen from '../app/screens/details';
 import renderer from 'react-test-renderer';
 

describe(' tests', () => {
    it('Render the component correctly ', () => {
    
        const route = {
            params:{        
               data:{"body": "est rerum tempore vitae", "id": 2, "title": "qui est esse", "userId": 1}
            }
        }
        const navigate = jest.fn();
        const tree = renderer.create(<DetailesScreen route= {route} />).toJSON();
        expect(tree).toMatchSnapshot();
      });
    });