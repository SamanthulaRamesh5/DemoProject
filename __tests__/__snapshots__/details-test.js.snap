// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` tests Render the component correctly  1`] = `
<View
  style={
    Object {
      "backgroundColor": "gray",
      "borderRadius": 10,
      "margin": 10,
      "padding": 5,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "fontWeight": "bold",
        "textTransform": "capitalize",
      }
    }
  >
    qui est esse
  </Text>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 16,
      }
    }
  >
    est rerum tempore vitae
  </Text>
  <Text />
</View>
`;
